### ISSUE TITLE
Support upgrading of existing repositories to ODS

### ISSUE THREAD

**Issue by clemensutschig** [opendevstack/ods-project-quickstarters#109]
Today One needs to start with a Quickstarter- which creates the full delivery chain. It should be possible to upgrade an existing Repo and just add ocp artifacts and a jenkinsfile.

**Comment by michaelsauter** [comment]
Yes, that would be great. How do you think this "upgrading" should work exactly? Would you ask users to select a repo and one of the quickstarters? Because just having a repo is not enough - we need to know what "kind" of project it is in order to create a sensible Jenkinsfile and OCP resources. E.g. one has an existing repo with a Node project inside, and we would then create the OCP resources, Jenkinsfile etc. as they are in the `node-express` quickstarter ...

**Comment by clemensutschig** [comment]
@michaelsauter  - here is how I envision it.
you pick a quickstarter (so we know the type) - and the name == an existing repo INSIDE your bitbucket project. 

there would be specific upgrade quickstarters (?) - that ensure that the prov app does not try to create a new bitbucket repo, and the quickstarter would only add the missing ocp / jenkinsfile / dockerfile pieces.

I took a quick look into the codebase _ and that's doable with reasonable effort. Even if the first version would only be to add a generic jenkinsfile - at least all the pain with ocp resources ... is gone. and then the user only needs to amend the jenkinsfile to his/her tech being used.
